tests/integration/cassandra_test.go

```diff
@@ -1,160 +0,0 @@
-// The MIT License
```

Comment:
@wxing1292 My understanding is those tests are covered in `./common/persistence/tests`. So I can remove the tests from here?

Reply:
if there are no tests referencing these setup, feel free to delete